css/index.css

```diff
@@ -104,14 +166,53 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+.box:nth-of-type(1) {
+    background: teal;
+}
+
+.box:nth-of-type(2) {
+    background: gold;
+}
+
+.box:nth-of-type(3) {
+    background: cadetblue;
+}
+
+.box:nth-of-type(4) {
```

Comment:
Nice solution, you specified the color of the boxes using the "nth-of-type" CSS selector. Another way to achieve the same result is to keep the "box" class that assigns that specific element its shape and add an additional class to each box to specify the color of the box. For example:

```
<div class="boxes">
    <div class = "box box1">Box 1</div>
    <div class = "box box2">Box 2</div>
    <div class = "box box3">Box 3</div>
</div>
```

Great job